core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1504,7 +1504,7 @@ private void deleteFromEntry(DeleteFileEntry entry) {
               failedToDelete = !ufsDeleter.delete(alluxioUriToDel, delInode);
             }
           } catch (InvalidPathException e) {
-            LOG.warn(e.getMessage());
+            LOG.warn("Failed to delete path from UFS: {}", e.getMessage());
```

Comment:
Will `e.getMessage()` already include the UFS path?

Reply:
This exception is thrown when ufsDeleter failed to resolve mount table as the alluxio URL is invalid.
thus, it is about alluxio URL which will be included in exception message